DataFormats/METReco/src/classes_def.xml

```diff
@@ -169,9 +169,16 @@
   </class>
   <class name="edm::Wrapper<reco::HcalHaloData>"/>
 
+  <class name="HaloTowerStrip" ClassVersion="2">
+   <version ClassVersion="2" checksum="665188928"/>
+  </class>
+
+  <class name="std::vector<HaloTowerStrip>"/>
+  <class name="edm::Wrapper<std::vector<HaloTowerStrip> >"/>
+
   <class name="reco::CSCHaloData" ClassVersion="11">
```

Comment:
The comment is about CSCHaloData block. It now reads as
```diff
   <class name="reco::CSCHaloData" ClassVersion="11">
-   <version ClassVersion="11" checksum="1723373351"/>
    <version ClassVersion="10" checksum="3522217120"/>
+   <version ClassVersion="11" checksum="1723373351"/>
   </class>
```
There should be no change in this block instead (it's a fairly trivial move of one line, but it will show up in file commit history as a change related to this PR, which it should be)